src/PhpGenerator/Method.php

```diff
@@ -392,4 +404,23 @@ public function setNamespace(PhpNamespace $val = NULL)
 		return $this;
 	}
 
+	/**
+	 * @param  string|NULL
+	 * @return self
+	 */
+	public function setReturnType($val)
```

Comment:
In fact, setVisiblity(NULL) converts it to empty string. There are inconsistencies.